tensorflow_similarity/augmenters/barlow.py

```diff
@@ -45,21 +107,41 @@ def augment(
         num_augmentations_per_example: int = 2,
         is_warmup: bool = True,
     ) -> List[Any]:
-
+ 
         with tf.device("/cpu:0"):
             inputs = tf.stack(x)
-            inputs = tf.cast(inputs, dtype="float32") / 255.0
+            inputs = tf.cast(inputs, dtype="float32") / 255
             views = []
-
+ 
+            #NOTE: do i need to normalize here? or before
```

Comment:
This is tricky as most images should be uint8 255, but that's not always true. If the user already called an image transform function, then we definitely wouldn't want to normalize here. I'd likely avoid this step as I don't think any of the tf.image functions require the images to be in a specific range.

Reply:
I believe solarization does require normalization - i think its a good idea to ask if the user wants normalization via a parameter in the constructor and apply it accordingly.

Reply:
Something that I had noticed was that when normalizing in the augmenter fn(ex. augment_barlow), the model has huge losses in performance which confuses me.

Reply:
BTW, something that I had noticed was that when normalizing in the augmenter fn(ex. augment_barlow), the model has huge losses in performance which confuses me.

Reply:
Okay, i've made normalization optional so that it can be enabled/disabled. Also, have changed augmentation_utils.solarize so that it can accept non-normalized images.